source/templates.tex

```diff
@@ -1719,6 +1720,47 @@
 \end{codeblock}
 \end{example}
 
+\rSec3[temp.constr.fold]{Fold expanded constraint}
+
+\pnum
+A \defnadj{fold expanded}{constraint} is formed from a constraint $C$ and
+a \grammarterm{fold-operator}
+which can either be \tcode{\&\&} or \tcode{||}.
+A fold expanded constraint is a pack expansion\iref{temp.variadic}.
+Let $N$ be the number of elements
+in the pack expansion parameters\iref{temp.variadic}.
+
+\pnum
+A fold expanded constraint whose \grammarterm{fold-operator} is \tcode{\&\&}
+is satisfied if it is a valid pack expansion and
+if $N = 0$ or if for each $i$ where $0 \le i < N$ in increasing order,
+$C$ is satisfied
+when replacing each pack expansion parameter
+with the corresponding $i$-th element.
```

Comment:
we typically use the syntax `$i^\text{th}$` for this

Reply:
fixed